core/new-gui/src/app/dashboard/component/feature-container/user-project-list/user-project-list.component.ts

```diff
@@ -51,20 +51,26 @@ export class UserProjectListComponent implements OnInit {
     this.userProjectEntriesIsEditingName = this.userProjectEntriesIsEditingName.filter(index => index != pid);
   }
 
-  public saveProjectName(project: UserProject, newName: string, index: number): void {
+  public saveProjectName(project: UserProject, newName: string): void {
     // nothing happens if name is the same
     if (project.name === newName) {
       this.removeEditStatus(project.pid);
     } else if (this.isValidNewProjectName(newName, project)) {
       this.userProjectService
       .updateProjectName(project.pid, newName)
       .pipe(untilDestroyed(this))
-      .subscribe(() => {
-        this.removeEditStatus(project.pid);
-        this.getUserProjectArray(); // refresh list of projects, name is read only property so can"t edit
-      });
+      .subscribe(
+        () => {
+          this.removeEditStatus(project.pid);
+          this.getUserProjectArray(); // refresh list of projects, name is read only property so cannot edit
+        },
+        (err: unknown) => {
+          // @ts-ignore
+          this.notificationService.error(err.error.message);
+        }
+      );
```

Comment:
just curious, have you tested this error handler will be triggered? since I thought the non-200 code will break the observable thus has to be handled in the service instead. I might be completely wrong though

Reply:
yes, I tested it before pushing and the error handler does get triggered.  I believe for non-200 codes the Observable will throw the error and then be in an error state with its lifecycle being over